src/OpenTelemetry.Exporter.Geneva/TLDExporter/TldLogExporter.cs

```diff
@@ -248,7 +240,12 @@ internal void SerializeLogRecord(LogRecord logRecord)
         // Part A - ex extension
         if (logRecord.Exception != null)
         {
-            eb.AddCountedAnsiString("ext_ex_type", logRecord.Exception.GetType().FullName, Encoding.UTF8);
+            var fullName = logRecord.Exception.GetType().FullName;
+            if (!string.IsNullOrEmpty(fullName))
+            {
+                eb.AddCountedAnsiString("ext_ex_type", fullName, Encoding.UTF8);
+            }
+
```

Comment:
for my learning, when would the Exception Type be null or empty?

Reply:
`Exception.Type` shouldn't be `null` but `Type.FullName` [may be null](https://learn.microsoft.com/dotnet/api/system.type.fullname). Why would `FullName` be `null`? Who knows! Probably some edge case thing not often seen like a `Type` dynmaically constructed at runtime? This change here makes it defensive should we ever run into it 🤷